<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1157</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="Group_Dolar_Bl_Of">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>110</y>
      <width>441</width>
      <height>461</height>
     </rect>
    </property>
    <property name="title">
     <string>Dolar</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Dolar Oficial:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Dolar Blue:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Dolar_Oficial_Compra">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>499.666</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Dolar_Oficial_Venta">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>80</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>499.666</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Compra:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Venta:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Dolar_Blue_Venta">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>170</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>499.666</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Dolar_Blue_Compra">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>170</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>499.666</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>170</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Venta:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Compra:</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>111</y>
       <width>401</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>401</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>401</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QTableWidget" name="Table_Cant_Dolares">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>276</y>
       <width>380</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="button_Guardar_Tabla_Dolar">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>420</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Guardar Tabla</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_Actualizar_Datos_Dolar">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>214</y>
       <width>91</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Actualizar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>130</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Total Blue Venta:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Total_en_Pesos_Dolar">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>250</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>499.666</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>18</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fecha Ultima Actualizacion :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Fecha_Ultima_Actualizacion">
     <property name="geometry">
      <rect>
       <x>234</x>
       <y>19</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>YYYY/MM/DD HH:MM:SS</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Group_UVA">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>438</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string>UVA</string>
    </property>
    <widget class="QPushButton" name="button_Ver_Historial_Pesos_Total">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>156</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Historial Pesos</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="Table_Cant_Uvas">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>196</y>
       <width>381</width>
       <height>201</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_Fecha_Actualizacion_UVA">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>66</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>YYYY/MM/DD HH:mm:ss</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_Ver_Historial_UVA">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>156</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Historial UVA</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>26</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Precio UVA: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Precio_UVA">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>26</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>$.....</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_Actualizar_UVA">
     <property name="geometry">
      <rect>
       <x>218</x>
       <y>22</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ACTUALIZAR VALOR UVA</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Total_en_Pesos">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>106</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>$.....</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>66</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fecha Ult. Actualizacion:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>106</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Total en Pesos:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_Guardar_Tabla_UVA">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>400</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Guardar Tabla</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>32</y>
      <width>461</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="button_Pestania_UVA">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>13</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>UVA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_Pestania_DOLAR">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>13</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DOLAR</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1157</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
